### PR TITLE
nodejs task1

### PR DESCRIPTION
**Pull request by ollyyasinski**
image

### CODE REVIEW

File: nodeJS/logger.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    new winston.transports.File({
+      filename: "logs/error.log",
+      level: "error"
+    }),
+    new winston.transports.File({ filename: "logs/combined.log" })
+  ]
+});
+
+// If we're not in production then log to the console
+if (process.env.NODE_ENV !== "production") {
```
Comment (GlebShaucou) [inline nodeJS/logger.js]: It is useful to know what was requested even on production.

File: nodeJS/logs/combined.log [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{"message":"Request method: GET , url: /news/2","level":"info","timestamp":"2019-11-27 15:30:39"}
```
Comment (GlebShaucou) [inline nodeJS/logs/combined.log]: Please, add this file to gitignore.

File: nodeJS/logs/error.log [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{"message":"400 - ID is not valid - /news/77 - GET - ::1","level":"error","timestamp":"2019-11-27 16:03:29"}
```
Comment (GlebShaucou) [inline nodeJS/logs/error.log]: Please, add this file to gitignore.

File: nodeJS/package.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "nodejs",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node index.js"
```
Comment (GlebShaucou) [inline nodeJS/package.json]: For development you could use **nodemon**

Comment (ollyyasinski) [reply nodeJS/package.json]: I've added. Thanks

File: nodeJS/logs/error.log (hunk 3 above)
Comment (ollyyasinski) [reply nodeJS/logs/error.log]: I've added as a working example. Removed now =)

File: nodeJS/logs/combined.log (hunk 2 above)
Comment (ollyyasinski) [reply nodeJS/logs/combined.log]: Removed

**Pull request merged**